domains/nxyy.json

```diff
@@ -4,6 +4,6 @@
         "email": "nxyylol@protonmail.com"
     },
     "record": {
-            "CNAME": "hosts.is-a.dev"
+        "CNAME": ""[{"type":"CNAME","value":"hosts.is-a.dev","allowMultiple":false}]""
```

Comment:
CNAME record should just be a simple string.

Reply:
I wasn't planning on using the hosting service, but I get it

Reply:
Oh wait, why are u editing my regular domain lol